test/resources/mnist/smdebug_mnist.py

```diff
@@ -30,6 +30,22 @@
 import numpy as np
 import random
 
+# Temporary patch this script until the MNIST dataset download issue get resolved
+# https://github.com/pytorch/vision/issues/1938
+import urllib
+# For python 2
+try:
+    class AppURLopener(urllib.FancyURLopener):
+        version = "Mozilla/5.0"
+
+    urllib._urlopener = AppURLopener()
+except AttributeError:
+    # For python 3
+    opener = urllib.request.build_opener()
+    opener.addheaders = [('User-agent', 'Mozilla/5.0')]
+    urllib.request.install_opener(opener)
+
```

Comment:
Can you submit this patch as a separate PR and remove it from the current PR? It is required for the SMDebug tests to pass in other PRs as well?